lambda/templates/create_db_users_lambda.json.tpl

```diff
@@ -0,0 +1,25 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "logs:CreateLogStream",
+        "logs:PutLogEvents"
+      ],
+      "Resource": [
+        "arn:aws:logs:eu-west-2:${account_id}:log-group:/aws/lambda/tdr-checksum-${environment}",
+        "arn:aws:logs:eu-west-2:${account_id}:log-group:/aws/lambda/tdr-checksum-${environment}:log-stream:*"
+      ]
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "ec2:CreateNetworkInterface",
```

Comment:
Actions not is alphabetical order: 

- `"ec2:CreateNetworkInterface"`
- `"ec2:DeleteNetworkInterface"`
- `"ec2:DescribeNetworkInterfaces"`

Reply:
I've switched them around :+1: